apps/yapms/src/routes/app/[country]/[map]/[year]/initialize/LoadRegions.ts

```diff
@@ -47,7 +47,8 @@ function editRegion(regionID: string) {
 	const region = get(RegionsStore).find((region) => region.id === regionID);
 	EditRegionModalStore.set({
 		region: region ?? null,
-		open: region !== undefined
+		open: region !== undefined,
+		confirmDisabled: false
```

Comment:
I don't think this needs to be controlled globally from a store, it could be controled inside the modal itself.